### ISSUE TITLE
Error: Failed to Delete Message

### ISSUE THREAD

**Issue by jhadenfeldt** [ChangemakerStudios/Papercut#91]
I am not able to delete the Welcome-message directly after installing the program:

![image](https://user-images.githubusercontent.com/959356/35130463-d9ea83e8-fc75-11e7-87c9-287aa2993129.png)

I installed it without administrator permissions and used the default settings during the install process.

**Comment by jijiechen** [comment]
It seems this message is a builtin message, thus you shouldn't delete it.

**Comment by LaughingJohn** [comment]
First time using Papercut and I noticed the same thing. First thing I did was try and delete the message.

When it starts it says this in the log:

[Information] Loading Messages from the Following Path(s) ["C:\Users\<USER>\AppData\Roaming\Papercut", "C:\Program Files (x86)\Changemaker Studios\Papercut\", "C:\Program Files (x86)\Changemaker Studios\Papercut\Incoming"]

Is there any way of preventing it using any folders under Program Files? I couldn't find anything in the config other than the appdata folder..
